### DISCUSSION TITLE
Setup parameters in Epever

### DISCUSSION THREAD

**Discussion by PrebenPayne** [softwarecrash/EPEver2MQTT#17] [Q&A]
Hello, is it possible to set up the parameters internally in Epever via Epever2mqtt?

**Comment by softwarecrash** [comment]
actual no
only load control and set time

**Reply by PrebenPayne** [reply]
Ok, thank you. And thanks for a nice application and the good work 😊.
If you later plan to update the application, so that these parameters can be written to the Epever, that would be nice 😊, and then it is possible to completely avoid Epevers own setup page or setup via the front panel.
I think your approach via mqtt is better than controlling the Epever by their own method. So thank you

**Reply by softwarecrash** [reply]
what you want to control? the epever have a lot of configuration settings, not all make sense to make it editable for some users.
actual i dont have this implemented by security, when a user click the wrong thing, all can burn ;)

you can make a feautere request with description what you want to control, interface look etc

**Reply by PrebenPayne** [reply]
I want to be able to setup all parameters for charging/discharging limits, battery type, all those parameters that can be set if you connect to your Epever via RS485 and use Epevers own application. But I think controlling it all from mqtt and Home Assistant is a better solution

**Reply by nobbystyles** [reply]
BATTERY_SOC is an item which indicates values almost much to high (Tracer 5420). No chance to correct the value?

**Reply by softwarecrash** [reply]
> BATTERY_SOC is an item which indicates values almost much to high (Tracer 5420). No chance to correct the value?

these value is based on the battery type and calculation from the epver device, set your battery type right. there is no way to correct this.

**Reply by PrebenPayne** [reply]
I think I have noticed for the Epever that it is the floating voltage setting that affects the SOC. Setting float voltage higher will decrease the SOC

**Reply by nobbystyles** [reply]
Although I think a have appropriate settings I understand that there is nothing that can be done. My settings just for information below. Thank you.
BatType: 0
BatCap: 280
TempComp: -3
OVCutOff: 29,00
ChargLmt: 28,00
OVRcv: 27,00
EquilV: 27,40
RaisV: 27,40
FloCharg: 27,00
RaisRcv: 26,90
LVRcv: 22,00
WarnRcv: 25,00
LVWarn: 24,00
LVCutOff: 21,00
DischaLmt: 20,00
RatedLv: 1
EquilDur: 1
RaisDur: 15
BatDisch: 30
BatCharg: 80
ChargMode: 0
LiBatteryProtection2: 0
DropPower: 1
Controlsignal: 1

**Reply by PrebenPayne** [reply]
Try to set FloCharg to 27.50 then I think SOC will drop

**Reply by nobbystyles** [reply]
Trying to set the Float_Volts to 27.5 via MQTT is not accepted in my case. First the new value turns red then it reverts to 27.

**Reply by PrebenPayne** [reply]
Ok, probably because there are internal rules for the settings, meaning maybe Float volts cannot be higher than EquilV: 27,40 and RaisV: 27,40. So maybe setting to 27.4 or 27.3 is allowed

**Reply by nobbystyles** [reply]
Same behaviour - as you wrote I think internal rules do their work, too ... it is not a real problem because I can look up (and change) the SOC in my "Daly2MQTT-device (BMS)" and take this value as reference.
